tests/explainers/test_tree.py

```diff
@@ -821,9 +814,7 @@ def test_provided_background_independent():
 
     np.random.seed(10)
 
-    X, y = shap.datasets.iris()
-    X = X[:100]
-    y = y[:100]
+    X, y = shap.datasets.iris(n_points=100)
```

Comment:
I believe this is the change that is causing an extra test failure. The iris dataset Y labels contain three unique values in ascending order : `[0, 0, 0, ..., 1, 1, 1, ..., 2, 2, 2, ...]`

If you slice the first 100 rows, one ends up with just 0s and 1s. However, with a random sample of 100 rows, one ends up with 0s, 1s and 2s, and the XGBoost model raises an Exception.

So suggest reverting to original behaviour:

```suggestion
    X, y = shap.datasets.iris()
    # Select the first 100 rows, so that the y values contain only 0s and 1s
    X = X[:100]
    y = y[:100]
```

Reply:
Thanks!